### PR TITLE
Implement notifications endpoint on top of Django + Docker

### PR DESCRIPTION
**Pull request by iprunache**
## Why
We need to create a real backend for the mobile app to poll for the latest notification. On the long term the mobile app needs to be able to query the backed for full data on notifications since data that can be sent with push notifications may be limited.

## What
- [x] setup UI microservice REST API using Django + Tastypie
- [x] build Docker image with the UI microservice
- [x] sync host folder with container deploy folder for dev purposes
- [x] create notifications endpoint
   - [x] add filter for retrieving the latest notification
   - [x] add `since` filter for filtering notifications that are newer than a specified timestamp

### CODE REVIEW

**Comment by iprunache** [comment]
@aismail @mozgovipc If you've got the time, please review this.

I'm using iso8601 compliant dates in the API output which I find is better than using UNIX timestamps for human readability but that can be changed with one line of code. I am aware of the recommendation for using timestamps but I'd like to understand the reasons behind that decision better before switching.

**Comment by aismail** [comment]
@iprunache the reason for using unix timestamps is because it doesn't require any complex library on the receiving end to parse the dates. In the JS world especially, this is a difficult and painful and buggy subject. Maybe to have the best of both worlds, we just deliver all timestamps in both formats?

File: README.md [hunk 1]
```diff
@@ -73,7 +75,7 @@ To run the container you first need to have a running `cami-store` container for
 Then run the `cami-medical-compliance` container linking it to the `cami-store` container.
 ``
 
-# Setup using Vagrant
+# Setup using Vagrant(this is broken right now and might not get revived!)
```
Comment (aismail) [inline README.md]: :)

File: docker-compose.yml [hunk 2]
```diff
@@ -19,3 +19,29 @@ services:
     volumes:
       - .:/cami-project
     entrypoint: ./docker/lib/wait-for-it/wait-for-it.sh -s -t 30 cami-store:3306 -- docker-entrypoint.sh
+
+  # Run migrations in a separate container from the main applications. The
```
Comment (aismail) [inline docker-compose.yml]: Thank you for coming up with this solution!

File: frontend/api/resources.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+from models import Notification
+
+from django.utils.timezone import is_naive
+
+# Get an instance of a logger
+logger = logging.getLogger('django')
+
+
+class TZAwareDateSerializer(Serializer):
```
Comment (aismail) [inline frontend/api/resources.py]: @mozgovipc here's how to turn the dates into UNIX timestamps on the ListPremier API

File: frontend/frontend/urls.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Including another URLconf
+    1. Import the include() function: from django.conf.urls import url, include
+    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import include, url
+from django.contrib import admin
+
+urlpatterns = [
+    url(r'^api/', include('api.urls')),
+    url(r'^admin/', admin.site.urls),
```
Comment (aismail) [inline frontend/frontend/urls.py]: @iprunache I'm not sure whether the admin interface is needed in the URL routes here. What would the use-case be at the API level?

Comment (iprunache) [inline frontend/frontend/urls.py]: @aismail I'm not sure I understand the comment. Should we disable the admin altogether? I have no idea what it's good for.

Comment (aismail) [inline frontend/frontend/urls.py]: @iprunache the admin is just a pretty frontend on top of the MySQL db. Something like PHPMyAdmin but a bit better :) That works on top of Django entities. So yeah, I'd remove it.

**Pull request merged**